.env.sample

```diff
@@ -87,4 +87,6 @@ KEYCLOAK_PUBLIC_KEY_PATH = path to the pem/secret file
 ORG_ID_HEADER_NAME = Org-id                                                      // Organisation id header name
 ADMIN_TOKEN_HEADER_NAME = admin-access-token                                     // admin access token header name
 ADMIN_ACCESS_TOKEN = ivopeiovcie-----------lvkkdvkdm                             // admin access token
-APP_PORTAL_DIRECTORY='/ml/'                                                      // App portal directory
\ No newline at end of file
+APP_PORTAL_DIRECTORY='/ml/'                                                      // App portal directory
+
+AUTH_CONFIG_FILE_PATH='config.json'                                              // Auth config file path
```

Comment:
@priyanka-TL, please sample config.json with existing SaaS QA values. Call it config.sample.json